tests/portal/test_registration.py

```diff
@@ -20,11 +20,16 @@ def test_rhi_register(self):
         self.vm.rhsm_register(distro='rhel7')
         self.vm.register_to_insights()
         self.host_name = self.vm.hostname.rstrip('\n')
+
         # UI tests for checking registered system appeared on customer portal UI
-        self.login.login_to_portal()
-        self.search = UISearch(self.driver)
-        self.result_text = self.search.register_system(self.host_name)
-        self.assertIn(self.host_name, self.result_text)
+        try:
+            self.login.login_to_portal()
+            self.search = UISearch(self.driver)
+            self.result_text = self.search.register_system(self.host_name)
+            self.assertIn(self.host_name, self.result_text)
+        except Exception as e:
```

Comment:
In this case, your test case will never fail as it will catch an exception, and let the test pass every time. Correct?